src/NumericalAlgorithms/Spectral/Filtering.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+/// \cond
+class Matrix;
+template <size_t>
+class Mesh;
+/// \endcond
+
+namespace Spectral {
+/*!
+ * \ingroup SpectralGroup
+ * \brief Matrices for filtering spectral coefficients.
+ */
+namespace filtering {
+/*!
+ * \brief Returns a `Matrix` by which to multiply the modal coefficients to
+ * apply a stable exponential filter.
```

Comment:
The .cpp and the test both suggest this is returning a matrix to multiply the *nodal* coeffs.

I suggest changing modal -> nodal on line 21 (because in the brief I just want to know how to use the matrix), and also changing "coefficients" -> "modal coefficients" on line 24 below (because the more detailed discussion below should precisely define the filter in terms of its action on modal coeffs).